Sources/StreamChatUI/CommonViews/AvatarView/ChatChannelAvatarView.swift

```diff
@@ -25,13 +31,27 @@ open class ChatChannelAvatarView: _View, ThemeProvider, SwiftUIRepresentable {
     open var imageMerger: ImageMerging = {
         DefaultImageMerger()
     }()
+    private lazy var imageProcessor: ImageProcessor = {
+        return NukeImageProcessor()
+    }()
+    private lazy var imageCDN: ImageCDN = {
+        return StreamImageCDN()
+    }()
 
+    // MARK: - Layout
     override open func setUpLayout() {
         super.setUpLayout()
         embed(presenceAvatarView)
+        embed(shimmerView)
+        shimmerView.isSkeletonable = true
     }
 
     override open func updateContent() {
+        loadIntoAvatarImageView(from: nil, placeholder: nil)
+        shimmerView.layer.cornerRadius = avatarCornerRadius
+        shimmerView.skeletonCornerRadius = Float(avatarCornerRadius)
+        shimmerView.showAnimatedGradientSkeleton()
+        shimmerView.isHidden = true
```

Comment:
As we are having `shimmerView.isHidden = true`  do we need show Animated Gradient here  `shimmerView.showAnimatedGradientSkeleton()` ?

Reply:
@mdhanif-simformsolutions  Yes , if we don't start shimmer effect here , shimmer effect will not working as aspected, Thanks